projected-resource-requirements.md

```diff
@@ -4,6 +4,31 @@ This document is intended to examine the expected storage, processing, and bandw
 
 ## Basics
 
+### Message Sizes
+
+Here we establish the overhead for various messages in the protocol.
+
+#### Base protocol message overhead
+
+We expect most messages to use the regular MESSAGE type packet from discovery v5 which has 39 bytes of overhead.
+
+#### TALKREQ/TALKRESP message overhead
+
+We expect each TALKREQ message to contain:
+
+- `request-id`: 4 bytes
+- `protocol-id`: 2 bytes
+- `payload`: variable
+
+Combined with the base overhead from the message packet this gives us 55 bytes of overhead per packet.
```

Comment:
Nitpick but there are also going to be a few bytes of overhead from the rlp encoding of the message.